libs/server/src/lib/utils/stats.ts

```diff
@@ -147,7 +165,7 @@ export function calculateStatsFromChunks(cs: Chunk[]) {
 /* ------------------------------------------------------------------------------------------------------------------ */
 
 interface ChunkData {
-  id: number;
+  id: number | string;
```

Comment:
Why both?

Reply:
webpack comes back with both... but i can normalize to a string.

Reply:
Do it!